lib/TestBootstrap.lua

```diff
@@ -22,29 +25,21 @@ local function getPath(module, root)
 		table.insert(path, stripSpecSuffix(last.Name))
 		last = last.Parent
 	end
+	table.insert(path, stripSpecSuffix(root.Name))
 
 	return path
 end
 
 --[[
 	Find all the ModuleScripts in this tree that are tests.
 ]]
-function TestBootstrap:getModules(root, modules, current)
+function TestBootstrap:getModules(root, modules)
 	modules = modules or {}
-	current = current or root
-
-	for _, child in ipairs(current:GetChildren()) do
-		if child:IsA("ModuleScript") and child.Name:match("%.spec$") then
-			local method = require(child)
-			local path = getPath(child, root)
-
-			table.insert(modules, {
-				method = method,
-				path = path
-			})
-		else
-			self:getModules(root, modules, child)
-		end
+
+	if isSpecScript(root) then
+		self:processScript(root, modules)
+	else
+		self:processFolder(root, modules)
 	end
```

Comment:
I think we should rename `processFolder` to `processChildren` and take it out of this else branch -- there's no reason why spec scripts inside spec scripts shouldn't work, even if it's kind of weird.